client/src/HomePage.js

```diff
@@ -0,0 +1,173 @@
+import React from 'react';
+import "../node_modules/bootstrap/dist/css/bootstrap.min.css";
+
+const HomePage = () => (
+<>   
+<nav className="navbar navbar-expand-lg navbar-light">
+  <div className="container">
+    <a className="navbar-brand mr-5" href="#"><img src="https://raw.githubusercontent.com/HarshCasper/AllNotes/2401a417beda2e1fb7a8f09d74c6c89dbed2ed85/design/assets/All-Notes--Monogram-1.png" height="80" width="80"/><img src="https://raw.githubusercontent.com/HarshCasper/AllNotes/2401a417beda2e1fb7a8f09d74c6c89dbed2ed85/design/assets/All-Notes-Draft-1.png" height="110" width="110"/></a>
+    <button className="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
+      <span className="navbar-toggler-icon"></span>
+    </button>
+    <div className="collapse navbar-collapse badge rounded-pill bg-purple ml-5" id="navbarNav">
+      <ul className="navbar-nav pl-5 ml-4">
+        <li className="nav-item">
+          <a className="nav-link active" aria-current="page" href="#"><h5>Why AllNotes ?</h5></a>
+        </li>
+        <li className="nav-item ml-5">
+          <a className="nav-link active" href="#"><h5>Features</h5></a>
+        </li>
+        <li className="nav-item ml-5">
+          <a className="nav-link active" href="#"><h5>Help</h5></a>
+        </li>
+        <li className="nav-item ml-5">
+          <a className="nav-link active" href="#"><h5>Login</h5></a>
+        </li>
+        <li className="nav-item ml-5">
+          <a className="nav-link active" href="#"><h5>Sign Up</h5></a>
+        </li>
+      </ul>
+    </div>
+  </div>
+</nav>
+   
+<div className="container-fluid mt-2 ml-3">
+   <div className="row">
+      <div className="col-lg-6">
+         <img src="https://user-images.githubusercontent.com/53649201/108606703-8ddf7580-73e1-11eb-9133-e9d62c623eab.PNG" height="400" width="600" />
+      </div>
+      <div className="col-lg-6 mt-5">
+         <h1 class="intro mr-4">The simplest way to find and organize notes.</h1>
+         <p class="intro-two ml-5 pl-5">AllNotes helps you capture idea and find them fast.</p>
+         <button className="btn btn-outline-info mt-3 py-2 px-5 rounded"><strong>Sign up for free</strong></button>
+         <p class="intro-three mt-2 pt-1">Already have an account ? <span class="text-pink">Log in.</span></p>
+      </div>
+   </div>  
+</div>
```

Comment:
Can you shift this to another component? Since we don't have the Login/Signup functionality ready, we can possibly integrate them when we have it ready. Instead you can have the `The simplest way to find and organize notes.
AllNotes helps you capture idea and find them fast.` content up there as placeholder.